docs/getting_started/installation/installation.rst

```diff
@@ -79,5 +79,7 @@ Building the documentation
 
 mBuild uses `sphinx <https://www.sphinx-doc.org/en/master/index.html>`_ to build its documentation. To build the docs locally, run the following while in the ``docs`` directory::
 
-    $ pip install -r requirements.txt
+    $ cd docs
+    $ conda create -f docs-env.yml
```

Comment:
shouldn't this be `conda env create -f docs-env.yml`?

Reply:
Good catch!